README.md

```diff
@@ -26,28 +38,28 @@ sudo npm i -g @antora/cli @antora/site-generator-default
 
 If antora installation succeed, `antora -v` will return antora's version.
 
-##### Clone repository and build documentation website.
+##### Build documentation website.
 
 ```bash
-git clone https://github.com/decidim/docs-base
-cd docs-base
 antora antora-playbook.yml
 ```
-
-Website source code will be create under ```docs-base/build/site/decidim``` folder. Serve ```index.html``` with any application or http server.
-
-```bash
-xdg-open build/site/decidim/index.html
-```
+Jump to [site overview](#site-overview)
 
 ### Docker
 
-You can check this locally and install with Docker.
+Clone the repository and launch
 
 ```bash
-git clone https://github.com/decidim/docs-base
-cd docs-base
 docker run -u $(id -u) -v $PWD:/antora:Z --rm -t antora/antora antora-playbook.yml
+```
+Jump to [site overview](#site-overview)
+
+## Site overview
```

Comment:
I add new section because it's related to `jump to` sentence in different installation type.
It's grouping the method to display website, because it's common for each installation.
I wanted to respect DRY philosophy